content/05.rule3.md

```diff
@@ -0,0 +1 @@
+## Rule 3: Choose an appropriate model for the data
```

Comment:
It gets confusing when we use the term model, because it can refer to the neural network params as well as architecture among other things. 

How about

"Choose an appropriate neural network architecture and data representation"?